### ISSUE TITLE
Start to write JSDoc comments

### ISSUE THREAD

**Issue by NWuensche** [Laverna/laverna#590]
Hello everybody,
I'm relatively new to Laverna but I can say that I like it really much. However, I would like to make the project clearer for every developer who's working on this, especially for new ones. For one of my first pull requests ( #527 ), I needed almost a week, just because I couldn't find the methods who caused the problem because of a missing documentary. Of course, this search for the right files, debugging and so on granted me a lot of insight into the project, but it was very hard to follow the right track. This is the reason I would love to start to make comments, that JSDoc ( http://usejsdoc.org/ ) can use to render a documentary for the project. I hope everyone can understand this decission and will also start to take care of this in their next PRs and maybe restructure some existing .js files for JSDoc.
Thanks for the attention!

**Comment by JerJohn15** [comment]
@NWuensche I like the idea. It would be great if for a future feature, we could get some documentation made with a basic breakdown of the project structure (maybe include it in the wiki section).
